src/perf/lib/ThroughputClient.cpp

```diff
@@ -0,0 +1,419 @@
+/*++
+
+    Copyright (c) Microsoft Corporation.
+    Licensed under the MIT License.
+
+Abstract:
+
+    QUIC Perf Throughput Client Implementation.
+
+--*/
+
+#ifdef QUIC_CLOG
+#include "ThroughputClient.cpp.clog.h"
+#endif
+
+#ifndef _KERNEL_MODE
+#define QUIC_TEST_APIS 1
+#endif
+#include "msquichelper.h"
+#include "ThroughputClient.h"
+#include "ThroughputCommon.h"
+
+ThroughputClient::ThroughputClient(
+    ) {
+    QuicZeroMemory(&LocalIpAddr, sizeof(LocalIpAddr));
+    if (Session.IsValid()) {
+        Session.SetAutoCleanup();
+    }
+}
+
+QUIC_STATUS
+ThroughputClient::Init(
+    _In_ int argc,
+    _In_reads_(argc) _Null_terminated_ char* argv[]
+    ) {
+    Port = THROUGHPUT_DEFAULT_PORT;
+    TryGetValue(argc, argv, "port", &Port);
+
+    const char* Target;
+    if (!TryGetValue(argc, argv, "target", &Target)) {
+        WriteOutput("Must specify '-target' argument!\n");
+        return QUIC_STATUS_INVALID_PARAMETER;
+    }
+
+    uint16_t ip;
```

Comment:
Let's fix up the casing for these variable to match our style. I know you probably copied from quicping and it has the issue already...